client/src/component/Button/index.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Button } from '@material-ui/core';
+
+const IconLabeledButton = ({ variant, color, icon, text, event, disable }) => (
+  <Button
+    variant={variant}
+    color={color}
+    endIcon={icon}
```

Comment:
we need to starticon and endicon so please add it as props.
please see the [Link](https://material-ui.com/components/buttons/#buttons-with-icons-and-label)

Reply:
done